samtranslator/schema/aws_serverless_api.py

```diff
@@ -217,6 +217,7 @@ class Resource(BaseModel):
     Properties: Properties
     Condition: Optional[PassThrough]
     DeletionPolicy: Optional[PassThrough]
+    Connectors: Optional[PassThrough]
```

Comment:
We support `Connectors` on non-SAM resources as well, so there might be a more generic way of supporting this property. Perhaps allow additional properties (using `LenientBaseModel`), and use another schema to only validate `Connectors` on any resource with `Type` of `AWS::*::*`. No need to do in this PR; from top of my head I'm not entirely sure how to do this. Perhaps the existing [`any_cfn_resource`](https://github.com/aws/serverless-application-model/blob/af76a9e9158dcb30a52df650c6587d1546e550a6/samtranslator/schema/any_cfn_resource.py#L7-L9) gives inspiration.

Reply:
So we should have [schema validation test](https://github.com/aws/serverless-application-model/blob/a4b4b9ac4e720107b4af3635de0a4fc99feb3ad7/tests/schema/test_validate_schema.py#L85-L94) that makes sure a non-SAM resource with `Connectors` and e.g. integer `Id` fails validation.